op-e2e/eip4844_test.go

```diff
@@ -50,13 +51,37 @@ func testSystem4844E2E(t *testing.T, multiBlob bool) {
 		maxL1TxSize = derive.FrameV0OverHeadSize + 100
 		cfg.BatcherMaxL1TxSizeBytes = uint64(maxL1TxSize)
 	}
+	cfg.DataAvailabilityType = daType
 
 	genesisActivation := hexutil.Uint64(0)
 	cfg.DeployConfig.L1CancunTimeOffset = &genesisActivation
 	cfg.DeployConfig.L2GenesisDeltaTimeOffset = &genesisActivation
 	cfg.DeployConfig.L2GenesisEcotoneTimeOffset = &genesisActivation
+	cfg.DeployConfig.L1GenesisBlockBaseFeePerGas = (*hexutil.Big)(big.NewInt(7000))
+
+	// For each test we intentionally block the batcher by submitting an incompatible tx type up
+	// front. This lets us test the ability for the batcher to clear out the incompatible
+	// transaction. The hook used here makes sure we make the jamming call before batch submission
+	// is started, as is required by the function.
+	jamChan := make(chan error)
+	jamCtx, jamCancel := context.WithTimeout(context.Background(), 20*time.Second)
+	action := SystemConfigOption{
+		key: "beforeBatcherStart",
+		action: func(cfg *SystemConfig, s *System) {
+			driver := s.BatchSubmitter.NewTestDriver()
+			err := driver.JamTxPool(jamCtx)
+			require.Nil(t, err)
```

Comment:
```suggestion
			require.NoError(t, err)
```
better failure messages...

Reply:
done